From a1df1a6926ba70e2ab580c44d2796fa542379c8b Mon Sep 17 00:00:00 2001
From: Richard Bradley <richard.bradley@joyent.com>
Date: Wed, 8 May 2019 15:24:02 +0100
Subject: [PATCH] MANATEE-380 would like `manatee-adm promote` MANATEE-393
 manatee-adm does not listen for PG errors after connecting MANATEE-417
 Manatee should not use the master branch for its state machine dependency
 Reviewed by: David Pacheco <dap@joyent.com> Approved by: David Pacheco
 <dap@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 7064eaf..6d67b4d 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "carrier": "0.2.0",
-        "manatee": "git+https://github.com/joyent/manatee.git#33b33433070e1c83622ca6a555b4bc38f1eec0e6"
+        "manatee": "git+https://github.com/joyent/manatee.git#6e802583f517e74f8e6c1ccab26b01ce2bbbf8f9"
     },
     "sdcDependencies": {
         "config-agent": ">=1.2.0"
-- 
2.21.0

